src/test/java/org/sasanlabs/service/vulnerability/pathTraversal/PathTraversalTest.java

```diff
@@ -0,0 +1,306 @@
+package org.sasanlabs.service.vulnerability.pathTraversal;
+
+import org.junit.jupiter.api.Test;
+import org.mockito.InjectMocks;
+import org.sasanlabs.service.vulnerability.bean.GenericVulnerabilityResponseBean;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.RequestEntity;
+
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+
+class PathTraversalVulnerabilityTest {
```

Comment:
Please run `./gradlew spotlessApply` this will fix the indentation and will pass the CI/CD pipeline.

Reply:
![image](https://github.com/SasanLabs/VulnerableApp/assets/13570884/ff0c5956-1439-4269-91ec-907e3bd245a3)